README.md

```diff
@@ -52,26 +52,30 @@ Eventually, you should pass your token in the `Authorization` header for the rel
 
 ## Development
 
+The project provides scripts to manage the lifecycle of the application. If `nix` is available they will attempt to use it, otherwise it will fall back to the "raw" commands (i.e., using `stack build` directly).
```

Comment:
The user absolutely has the choice, I'll make it clearer that the scripts are optional.

Reply:
The script is optional sure, but it is the only documented approach. It would be nice to document both approaches and then mention that we have helper scripts to automate the choice

Reply:
This has been updated, could you check the README again and let me know if there's still anything you'd like me to include?